iceoryx_posh/include/iceoryx_posh/internal/popo/building_blocks/chunk_receiver_data.hpp

```diff
@@ -35,11 +36,14 @@ struct ChunkReceiverData : public ChunkQueueData
     /// we use one more than MAX_CHUNKS_HELD_PER_RECEIVER for being able to provide one new chunk
     /// to the user if they already have the allowed MAX_CHUNKS_HELD_PER_RECEIVER. But then the user
     /// has to return one to not brake the contract. This is aligned with AUTOSAR Adaptive ara::com
-    static constexpr uint32_t MAX_CHUNKS_IN_USE = MAX_CHUNKS_HELD_PER_RECEIVER + 1u;
+    static constexpr uint32_t MAX_CHUNKS_IN_USE =
+        ChunkQueueDataType::ChunkQueueDataProperties_t::MAX_CHUNKS_PER_RECEIVER + 1u;
```

Comment:
I thing I don't like from design point of view is that your ChunkQueueDataProperties include the MAX_CHUNKS_PER_RECEIVER. This is not used by the ChunkQueue itself but later in the class hierarchy by ChunkReceiver. Maybe this should be a separate config for ChunkReceiverData, or maybe add this single template parameter as having a config struct for one alone is a bit of overhead

Reply:
Yeah, got it. Put it now as separate param in chunk sender/receiver data. Have a look.